### PR TITLE
3241 vgap and vgap2 are not well named

### PR DESCRIPTION
**Pull request by ym1906**
## Description

It wasn't clear to me what these variables referred to exactly, therefore I have renamed them to 

vgap_xpoint_divertor (vertical gap between xpoint and divertor)
vgap_vv_thermalshield (vertical gap between vacuum vessel and thermal shield)

### CODE REVIEW

File: tests/regression/input_files/large_tokamak.MFILE.DAT.json
Comment (ajpearcey) [inline tests/regression/input_files/large_tokamak.MFILE.DAT.json:1]: Do we need this file in the regression? Has this slipped into the PR?

File: prep_sol_for_uq.ipynb
Comment (ajpearcey) [inline prep_sol_for_uq.ipynb:1]: Does this belong in the PR? Should it be in separate PR?

**Review by ajpearcey — CHANGES_REQUESTED** [review]
Overall looks good, but i think a few files slipped into the PR

File: tests/regression/input_files/large_tokamak.MFILE.DAT.json
Comment (ym1906) [reply tests/regression/input_files/large_tokamak.MFILE.DAT.json:1]: No, should have been removed!

File: prep_sol_for_uq.ipynb
Comment (ym1906) [reply prep_sol_for_uq.ipynb:1]: Thought I'd removed this one, will do it now

**Comment by ym1906** [comment]
@ajpearcey I removed those two files, let me know any other changes
